src/process.cr

```diff
@@ -1,60 +1,51 @@
-require "c/signal"
-require "c/stdlib"
-require "c/sys/resource"
-require "c/unistd"
+require "crystal/system/process"
 
 class Process
+  getter pid : Int64 = 0
+
+  # A pipe to this process's input. Raises if a pipe wasn't asked when creating the process.
+  getter! input : IO::FileDescriptor
+
+  # A pipe to this process's output. Raises if a pipe wasn't asked when creating the process.
+  getter! output : IO::FileDescriptor
+
+  # A pipe to this process's error. Raises if a pipe wasn't asked when creating the process.
+  getter! error : IO::FileDescriptor
+
+  # channel of process exit code
+  @waitpid : Channel(Int32) = Channel(Int32).new(1)
+  @wait_count = 0
+
+  protected def initialize(@pid)
+    wait_system
+    @wait_count = 0
+  end
+
+  # Open process by identifier
+  #
+  # Checks that process with given *pid* exists and is accessible.
+  def self.open(pid : Int64)
```

Comment:
There is no such method right now.

Reply:
Ok i removed it so we keep changes minimal and can move on.